anti-analysis/anti-av/check-for-sandbox-and-av-modules.yml

```diff
@@ -0,0 +1,40 @@
+rule:
+  meta:
+    name: check for sandbox and av modules
+    namespace: anti-analysis/anti-vm
```

Comment:
would you please fix the namespace here?

Reply:
Fixed, thanks for catching the typo.